journal.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+from pyramid.config import Configurator
+from pyramid.view import view_config
+from waitress import serve
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine
+
+Base = declarative_base()
+
+# add this import at the top:
+import datetime
```

Comment:
You should keep all your imports together at the top of a file.  Not here below where you set up `Base`.